Applications/Common-Platform/start-common.sh

```
@@ -0,0 +1,38 @@
+
+#!/bin/bash
+set -e
+
+# Set AMRIT_HOME robustly to the parent directory of this script
+AMRIT_HOME="$(dirname $(dirname $(realpath $0)))"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)